ndt7/download/measurer/measurer.go

```diff
@@ -77,10 +79,11 @@ func loop(
 			}
 			measure(&measurement, sockfp)
 			if sentConnectionInfo == false {
+				// TODO: eliminate ConnectionInfo from model.Measurement and send to client differently.
```

Comment:
Please, rather than a comment, create an issue for this task. I believe it should be possible to do that without breaking clients, but we should do that and see. Given that roll-out is happening soon, I want to make sure we do that (or not do that) before, so that we don't break stuff after. An issue seems better to ensure that.